packages/toggle-switch/toggle-switch.scss

```diff
@@ -3,207 +3,149 @@
 
 @use "./toggle-slider";
 
-@include jkl.light-mode-variables {
-    --jkl-toggle-switch-focus-outline-color: #{jkl.$color-focus-color};
```

Comment:
Disse endringene i CSS-variabler og klassenavn blir strengt talt en breaking change for CSSen.